src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const sourceObject = sourceArray.reduce((acc, curr) => {
+    const entry = curr.split(':').map((item) => item.trim());
+
+    acc[entry[0]] = entry[1];
```

Comment:
There is a potential issue where if the CSS property name is an empty string, it will still be added to the accumulator. Consider adding a check to ensure that both the property name and value exist before adding them to the object.